corpus/accounts/models.py

```diff
@@ -11,6 +11,7 @@
 from .validators import validate_phone_number
 from .validators import validate_reg_number
 from .validators import validate_roll_number
+from corpus.validators import validate_image
```

Comment:
nit: did you run the linter? Looks like this style would have been picked up by that.